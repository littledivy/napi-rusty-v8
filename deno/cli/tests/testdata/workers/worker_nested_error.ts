const worker = new Worker(
  new URL("worker_error.ts", import.meta.url).href,
  { type: "module", name: "baz" },
);
setTimeout(() => worker.terminate(), 30000);
