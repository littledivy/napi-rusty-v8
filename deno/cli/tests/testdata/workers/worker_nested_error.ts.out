[WILDCARD]error: Uncaught (in worker "bar") Error: foo[WILDCARD]
  throw new Error("foo");
        ^
    at foo ([WILDCARD]/workers/error.ts:[WILDCARD])
    at [WILDCARD]/workers/error.ts:[WILDCARD]
error: Uncaught (in worker "baz") (in promise) Error: Unhandled error event in child worker.
    at Worker.#pollControl ([WILDCARD])
error: Uncaught (in promise) Error: Unhandled error event in child worker.
    at Worker.#pollControl ([WILDCARD])
